fitz/helper-fields.i

```diff
@@ -471,10 +471,10 @@ void JM_get_widget_properties(fz_context *ctx, pdf_annot *annot, PyObject *Widge
         SETATTR_DROP(Widget, "field_type_string",
                 JM_UnicodeFromStr(JM_field_type_text(field_type)));
 
-        #if FZ_VERSION_MAJOR == 1 && FZ_VERSION_MINOR <= 22
+        #if FZ_VERSION_MAJOR == 1 && FZ_VERSION_MINOR < 22
```

Comment:
Are you sure this change is correct? We run daily tests with MuPDF 1.21, 1.22 and master (which has version 1.23), and they've been building and passing ok for a while. Maybe your mupdf master needs updating to get the change of its version to 1.23?

Reply:
I only did this because of compile errors.

Reply:
Great.